Source/PyromaniacIsFun/NeedPyromania.cs

```diff
@@ -179,7 +179,7 @@ public override void NeedInterval()
                 sb.Append(ExplanationFromObservedFire);
             }
 
-            sb.AppendLine($"\nFinal change rate: {gain * 100:+0;-0}% / day ({gain * 100 / GenDate.HoursPerDay:F0}% / hour)");
+            sb.AppendLine($"CF_PyromaniacIsFun_NeedPyrimania.FinalChangeRate".Translate((gain * 100).ToString("F0"), (gain * 100 / GenDate.HoursPerDay).ToString("F0")));
             lastDelta = gain * NeedTunings.NeedUpdateInterval / GenDate.TicksPerDay;
```

Comment:
Suggest remove `$`
Also suggest do `sb.AppendLine().AppendLine(...)` so that `CF_PyromaniacIsFun_NeedPyrimania.FinalChangeRate` doesn't have `\n` in the beginning.

Reply:
Applied